.github/ISSUE_TEMPLATE/blank-issue-template.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Blank issue template
```

Comment:
I wonder if people will just use this to bypass the templates? But I guess we would want to use this template ourselves?